tests/language/completion/test_mysql_naive_completion.py

```diff
@@ -0,0 +1,148 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import unittest
+from prompt_toolkit.completion import Completion
+from prompt_toolkit.document import Document
+from unittest.mock import Mock
+
+from ossdbtoolsservice.language.completion.mysqlcompleter import MySQLCompleter
```

Comment:
"from ossdbtoolsservice.language.completion.mysqlcompleter" this should come before "from prompt_...."